astropy/coordinates/tests/test_funcs.py

```diff
@@ -33,6 +33,38 @@ def test_sun():
     gcrs2 = get_sun(Time([northern_summer_solstice, equinox_2, northern_winter_solstice]))
     assert np.all(np.abs(gcrs2.dec - [23.5, 0, -23.5]*u.deg) < 1*u.deg)
 
+def test_sun_02():
+    """
+    Test `that get_sun` produces similar values to skyfield (de421).
```

Comment:
@jberlanga - to give you just a bit more detail here if you don't know:  double-backticks roughly mean "code sample", whereas single-backticks mean "link to some class or function with that name".  So in code that actually makes it into the docs, as @bsipocz said, you'd need to do something like `` `astropy.coordinates.get_sun` ``